app/controllers/import_controller.rb

```diff
@@ -189,6 +204,16 @@ def lab_result(data, row_ind)
     }
   end
 
+  def vaccination(data, row_ind)
+    {
+      group_name: import_field(:group_name, data[0], row_ind),
+      product_name: import_field(:product_name, data[1], row_ind),
+      administration_date: import_field(:administration_date, data[2], row_ind),
+      dose_number: import_field(:dose_number, data[3], row_ind),
+      notes: import_field(:notes, data[4], row_ind)
```

Comment:
I think these `import_field`s aren't working exactly as we want them, because in the `validation_helper`, we need to add the correct checks to the `checks` field for these elements:
```ruby
    administration_date: { label: 'Administration Date', checks: [] },
    dose_number: { label: 'Dose Number', checks: [] },
    notes: { label: 'Notes', checks: [] },
    group_name: { label: 'Vaccine Group', checks: [] },
    product_name: { label: 'Product Name', checks: [] }
```
As an example, right now something like `product_name` is case sensitive, which breaks the pattern we've followed with other `enum` fields.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/958/commits/ab7f19cf8ad4bd93f4d65ff43baf58e9f4c9c392